### PR TITLE
fix(curriculum): Learn Greetings in your First Day at the Office - Task 32 - Sentence doesn't match audio

### LINKED ISSUE #53010: Learn Greetings in your First Day at the Office - Task 32 - Sentence doesn't match audio
### Describe the Issue

The fill-in-the-blank sentence doesn't match what the character says.

### Affected Page

https://www.freecodecamp.org/learn/a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/task-32

### Your code

The written sentence is:
`Thanks, Maria, for ___ me around the ___.`

### Expected behavior

The character says:
`Thanks for ___ me around the ___, Maria.`

### Screenshots

_No response_

### System

- Device: all
- OS: all
- Browser: all
- Version: all

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by aakash-a-dev**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53010

I created a PR while setting it up my vs code deleted the file itself and my previous PR was mistake so created new PR because I had to delete my previous Fork repo due to technical glitch.
For solving this issue I followed the steps commented in the respective issue:

In https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a14b0e1c43d0ffabd15a9.md:
Change the text in the AUDIO REFERENCE tag from Thanks, Maria, for showing me the place! to Thanks for showing me around the place, Maria.
In https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md:
Change the text in the AUDIO REFERENCE tag from Thanks, Maria, for showing me around the place! to Thanks for showing me around the place, Maria.
Change the fill-in-the-blank sentence from Thanks, Maria, for _ me around the _. to Thanks for _ me around the _, Maria..

I hope for a Positive contribution forward.

### CODE REVIEW

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
change the English files please, not the Chinese files

**Comment by aakash-a-dev** [comment]
I have rectified the earlier misunderstanding I had. Sorry and Thank you for your feedback I rectified this time. I hope this time I am doing it correctly.

File: curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a14b0e1c43d0ffabd15a9.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: task-31
 
 <!--
 AUDIO REFERENCE:
-Tom: Everything looks great. Thanks, Maria, for showing me the place!
+Tom: Everything looks great. Thanks for showing me around the place, Maria.
```
Comment (ieahleen) [inline curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a14b0e1c43d0ffabd15a9.md:10]:
```suggestion
Tom: Everything looks great. Thanks, Maria, for showing me the place!
```

File: curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ dashedName: task-32
 
 <!--
 AUDIO REFERENCE:
-Tom: Thanks, Maria, for showing me around the place!
+Tom: Thanks for showing me around the place, Maria.
```
Comment (ieahleen) [inline curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md:10]:
```suggestion
Tom: Thanks, Maria, for showing me around the place!
```

File: curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ What did Maria do for Tom that he's expressing gratitude for?
       "startTime": 1,
       "finishTime": 3.8,
       "dialogue": {
-        "text": "Everything looks great. Thanks for showing me around the place, Maria.",
+        "text": "Everything looks great. Thanks, Maria, for showing me around the place!",
```
Comment (ieahleen) [inline curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md:70]:
```suggestion
        "text": "Everything looks great. Thanks for showing me around the place, Maria.",
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please remove the changes to the Chinese files, if the bot keeps answering they are still there.

**Review by aakash-a-dev — COMMENTED** [review]
As per the guidance I modified the code. Thank you

**Comment by aakash-a-dev** [comment]
It's showing 
GitHub - No translations via PRs / has-translation (pull_request_target) But I reverted to the original code

File: curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md (hunk 3 above)
Comment (ieahleen) [inline curriculum/challenges/chinese/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a18189b21f51037e518b4.md:70]:
```suggestion
        "text": "Everything looks great. Thanks for showing me around the place, Maria.",
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
please revert also this last change to the Chinese files

**Comment by aakash-a-dev** [comment]
Now I am crying but in Hope that this time I did the correction. Truly sorry for making you to review this code this much.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a14b0e1c43d0ffabd15a9.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ dashedName: task-31
 
 <!--
 AUDIO REFERENCE:
-Tom: Everything looks great. Thanks, Maria, for showing me the place!
+Tom: Everything looks great. Thanks, Maria, for showing me the place!.
```
Comment (gikf) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656a14b0e1c43d0ffabd15a9.md:10]:
```suggestion
Tom: Everything looks great. Thanks for showing me around the place, Maria.
```

**Review by gikf — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Comment by aakash-a-dev** [comment]
Maam @ieahleen Can you check it please, I had done and got two review approved as well.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍 

Note you will need to reset your main branch to be in sync with the freeCodeCamp:main branch. This is a reason for which it is highly suggested to not work on the main branch but to create feature branches.

**Pull request merged**